src/ui/components/VerifyPassword/VerifyPassword.test.tsx

```diff
@@ -82,65 +82,65 @@ describe("Verify Password on Cards Details page", () => {
     };
   });
 
-  // test.skip("It renders verify password when clicking on the big archive button", async () => {
```

Comment:
What is all this commented out code for?

Reply:
The PR is uncommenting the tests.

I mentioned this already in the channel that the previous PR to remove Aries would cause some tests to be commented out until this PR (and maybe the next) was merged in. Too many changes for one PR.